moduleroot/.travis.yml.erb

```diff
@@ -17,6 +17,9 @@ addons:
 <%   end -%>
 <% end -%>
 before_install:
+  - if [ $BUNDLER_VERSION ]; then
+      gem install -v $BUNDLER_VERSION bundler --no-rdoc --no-ri;
```

Comment:
We recently discovered  bundler 2 no longer accepts `--no-rdoc` and `--no-ri` in favor of `--no-document`. Apparently the latter two were deprecated a long time ago, but only in the help text.

Reply:
Thanks for the heads up @joshcooper :-)
When we drop Puppet 4 testing, we will be removing the bundler version pin and this code block therefore we should avoid running into any issues.